cosmoplots/axes.py

```diff
@@ -91,7 +91,10 @@ def change_log_axis_base(
         )
     return axes
 
-def figure_multiple_rows_columns(rows: int, columns: int) -> Tuple[Figure, List[Axes]]:
+def figure_multiple_rows_columns(rows: int, columns: int, 
+                                 labels: List[str] | None = None,
```

Comment:
The `|` type specifier was added in 3.10, so since cosmoplots supports 3.8 it should probably be `Union[List[str], None]`.